draft-ietf-quic-transport.md

```diff
@@ -5495,6 +5497,24 @@ Reason Phrase:
   This SHOULD be a UTF-8 encoded string {{!RFC3629}}.
 
 
+## HANDSHAKE_DONE frame {#handshake-done-frame}
+
+The server uses the HANDSHAKE_DONE frame (type=0x1e) to signal confirmation of
+the handshake to the client.  The HANDSHAKE_DONE frame contains no additional
+fields.
+
+This frame can only be sent by the server. Servers MUST NOT send a
+HANDSHAKE_DONE frame before completing the handshake.  A server MUST treat
+receipt of a HANDSHAKE_DONE frame as a connection error of type
+PROTOCOL_VIOLATION.
+
+A client MAY treat receipt of an acknowledgment for a packet it sent with 1-RTT
+keys as receipt of a HANDSHAKE_DONE frame.  This can be implemented by recording
```

Comment:
I misread this first sentence at first and got the wrong impression.  I think that this belongs in the other document, but it should probably instead say that:

> A client MAY consider the handshake to be confirmed when it receives an acknowledgement for a 1-RTT packet.  ...[the how to implement bit]

Reply:
Thanks. I applied your suggestion.